cpp/tests/CMakeLists.txt

```diff
@@ -168,13 +168,13 @@ ConfigureTest(INTEROP_TEST
 
 ###################################################################################################
 # - jit cache tests -------------------------------------------------------------------------------
-ConfigureTest(JITCACHE_TEST
-    "${CUDF_SOURCE_DIR}/src/jit/cache.cpp"
-    jit/jit-cache-test.cpp)
+# ConfigureTest(JITCACHE_TEST
+    # "${CUDF_SOURCE_DIR}/src/jit/cache.cpp"
+    # jit/jit-cache-test.cpp)
```

Comment:
Don't forget to enable them back (if they are not to be removed).

Reply:
Yikes! Good catch.